### PR TITLE
[KOGITO-2703] Add visa document link

### PR DESCRIPTION
**Pull request by ruromero**
Fix https://issues.redhat.com/browse/KOGITO-2703

* Add visa document link to process and UI
* Fix README markdown formatting
* Remove tabs and trailing spaces from index.html

### CODE REVIEW

**Review by tiagodolphine — APPROVED** [review]
LGTM, just the comment to add the screenshot where you can insert the visa document link.

**Comment by tiagodolphine** [comment]
@cristianonicolai do you have the link to the source for the model diagram? It would be great to have it in case any update is necessary? Ruben needed to edit the image to add the new visa attribute. Thanks.

**Comment by ruromero** [comment]
@tiagodolphine I have updated the image

**Comment by ruromero** [comment]
cc/ @mswiderski in case you want to take a look

**Review by cristianonicolai — APPROVED** [review]

File: kogito-travel-agency/basic/src/main/resources/META-INF/resources/index.html [hunk 1]
```diff
@@ -210,23 +223,27 @@ <h1>Travels</h1>
         $("#tasksModal").on('show.bs.modal', function (event) {
             var button = $(event.relatedTarget);
             var action = button.data('action');
-            var id = button.data('id');
+            var processInstanceId = button.data('id');
```
Comment (mswiderski) [inline kogito-travel-agency/basic/src/main/resources/META-INF/resources/index.html]: The whole point was to not refer to any process specific names as this is domain specific api

File: kogito-travel-agency/basic/src/main/java/org/acme/travels/Traveller.java [hunk 2]
```diff
@@ -22,13 +22,13 @@
 	private String email;
 	private String nationality;
 	private Address address;
+	private String visa;
```
Comment (mswiderski) [inline kogito-travel-agency/basic/src/main/java/org/acme/travels/Traveller.java]: So is this visa or visa application ? Looks more like visa application. If so would be more useful if the actual visa (from visa application) is set here thus coming from message event.

File: kogito-travel-agency/basic/src/main/resources/META-INF/resources/index.html (hunk 1 above)
Comment (ruromero) [reply kogito-travel-agency/basic/src/main/resources/META-INF/resources/index.html]: You're right. Renamed

File: kogito-travel-agency/basic/src/main/java/org/acme/travels/Traveller.java (hunk 2 above)
Comment (ruromero) [reply kogito-travel-agency/basic/src/main/java/org/acme/travels/Traveller.java]: I'll just rename it to visaApplication

**Pull request merged**